pkg/orchestrators/kubernetes.go

```diff
@@ -82,16 +83,22 @@ func (o *KubernetesOrchestrator) GetVolumes(volumeFilters volume.Filters) (volum
 			}
 
 			containers, _ := o.GetContainersMountingVolume(v)
-			if len(containers) > 0 {
-				v.HostBind = containers[0].HostID
-				v.Hostname = containers[0].HostID
-				v.Mountpoint = containers[0].Path
-			}
-
-			if b, _, _ := o.blacklistedVolume(v, volumeFilters); b {
-				continue
+			containerMap := make(map[string]bool)
```

Comment:
This variable is named the same as another one below. Could it be renamed to avoid confusion?